packages/redux/src/users/selectors.js

```diff
@@ -153,8 +153,7 @@ export const getPreferences = state => getEntity(state, 'preferences');
  *
  * @returns {boolean} Titles operation Loading status.
  */
-export const isTitlesLoading = state =>
-  getTitlesGetter(state.profile).isLoading;
+export const isTitlesLoading = state => getTitlesGetter(state.users).isLoading;
```

Comment:
Same thing as the plural comment. Please review everywhere for this matter 🙏

Reply:
Changed it 🙏